website/content/docs/concepts/adaptive-overload-protection/index.mdx

```diff
@@ -60,21 +58,24 @@ In some cases, a sudden influx of write requests that exceeds Vault's hardware
 capacity can result in the writes queueing for so long that every request times
 out before the write can make it through the queue. This makes Vault effectively
 unavailable to clients even though it is still processing requests and storing
-data as fast as it can. This is illustrated in the test results shown below for
-a workload of 100% logins.
+data as fast as it can. This can also result in Replication lagging behind causing
+data discrepancies between replicated clusters. This is illustrated in the test
+results shown below for a workload of 100% logins.
 
 ![Login workload telemetry graphs showing difference with and without adaptive overload protection for writes](/img/adaptive-overload-protection-writes.png)
 
 Adaptive Write Overload Protection prevents this scenario. It constantly
 monitors the current state of the write queue and uses a carefully tuned
 algorithm to allow just enough queueing to maximize throughput on the available
 hardware while keeping latencies under control and unnecessary rejections to a
-minimum.
+minimum. Writes related to replication are always handled, avoiding a sudden
+increase of write requests causing replication lag.
 
 Write overload protection was added in Vault Enterprise 1.17 as a beta feature
-which is disabled by default.
+which is now enabled by default for Raft Integrated Storage. Adaptive overload
+protection is disabled by default for Consul storage.
```

Comment:
```suggestion
and was disabled by default. Since Vault 1.18.0 it is enabled by default only when using Raft Integrated Storage. Adaptive overload
protection is disabled by default for Consul and other storage backends.
```